.jscs.json

```diff
@@ -1,4 +1,5 @@
 {
+	"preset": "./jscsrc-esnext",
```

Comment:
just as an example, you could also include rules, filters, etc; locally or from your dependencies